src/ServiceComposer.AspNetCore/ServiceComposer.AspNetCore.csproj

```
@@ -34,7 +34,7 @@
 
   <Target Name="AppVeyorPullRequestsTarget" AfterTargets="MinVer" Condition="'$(APPVEYOR_PULL_REQUEST_NUMBER)' != ''">
     <PropertyGroup>
-      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-PR.$(APPVEYOR_PULL_REQUEST_NUMBER).$(MinVerPreRelease)</PackageVersion>
+      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-pr.$(APPVEYOR_PULL_REQUEST_NUMBER).build.$(APPVEYOR_BUILD_NUMBER).$(APPVEYOR_BUILD_ID).$(MinVerPreRelease)</PackageVersion>
```

Come to think of it, `APPVEYOR_BUILD_NUMBER` is redundant, since it's usually going to be the same. It will change if another PR is merged to master while a PR is open, but it doesn't really add much value. I guess you could leave it in if you want, but you could change it to this, which also labels the value correctly as `build-id`:

```suggestion
      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-pr.$(APPVEYOR_PULL_REQUEST_NUMBER).build-id.$(APPVEYOR_BUILD_ID).$(MinVerPreRelease)</PackageVersion>
```

---

I like it, thanks!